condor_watch_q.py

```diff
@@ -216,6 +223,24 @@ def parse_args():
     return args
 
 
+def parse_unknown_args(unknown):
+
+    for unknown_arg in unknown:
+        print("Unknown command.", end=" ")
+        if unknown_arg.isdigit():
+            print("Did you mean condor_watch_q -clusters {} ?".format(unknown_arg))
+        elif "-totals" in unknown_arg:
+            print("Did you mean condor_watch_q -no-table?")
+        elif "-userlog" in unknown_arg:
+            print("Did you mean condor_watch_q -files FILE [FILE ...]?")
+        elif "-nobatch" in unknown_arg:
+            print("Defaults to batch, try condor_watch_q -groupby {batch,log,cluster}.")
+        elif "-" in unknown_arg:
+            print("condor_watch_q does not support the following functionality yet.")
```

Comment:
We don't need this anymore. Just leave off the generic check; argparse itself will catch unknown arguments, now that we're using a custom parser.